### DISCUSSION TITLE
Tested forwarding and all tests are fine except from tow domains using microsoft365 ?  show no-reply@forwardemail.net when reply

### DISCUSSION THREAD

**Discussion by vw-kombi** [forwardemail/forwardemail.net#235] [General]
I have no idea why this affects just these two senders.
I was using cloudflare email forwarding and as of Sunday, they were no longer delivering emails from these two domains.
They were failing with 'Gmail has detected that this message is4.7.0 suspicious due to the very low reputation of the sending domain'
This issue was affecting a few - nd all started Sunday - https://community.cloudflare.com/t/transient-error-421-4-7-0-when-forwarding-emails-to-gmail/629941
I then switched to forwardemail.net today and these same failures now are forwarded, but with no-reply@forwardemail.net as the reply to - unlike all other emails that are forwarded.
I have approached both customer, but they don't have an issue with anyone else (they think).  
They both use MS365 is one thing they have in common.

**Comment by vw-kombi** [comment]
Never mind - gave up on the forwarding and back to apaid for service.

**Comment by titanism** [comment]
Hi there, sorry for the delay.  If you see a `no-reply@forwardemail.net` it is known as a Friendly-From rewrite.

https://forwardemail.net/en/faq#how-do-you-handle-email-delivery-issues

See our dedicated section in the FAQ on this (link directly above) - this is required in order for email to pass DMARC, since the sender passed SPF but had no DKIM aligned signature to the From domain on the email, and if we forwarded it without rewriting it, then it would fail DMARC since the sender has `p=reject` policy.
